10x_scRNA/Snakefile

```diff
@@ -40,6 +40,7 @@ rule processSamples:
     shell: """
         module load cellranger
         cellranger count \
+            --create-bam true \
```

Comment:
This would require cellranger >= 8. Older versions would fail. So, (In theory...) now the pipeline only supports GEMX kits. However, I demultiplexed old chemistry kits with cellranger 8 just fine (10x docs imply the opposite). I'm only commenting for the record.

Reply:
Thanks for catching this. I would leave it for now, until it breaks, but for a cleaner solution we could probably grep the cellranger version before setting this flag.